samples/splitter/splitter.cpp

```diff
@@ -263,6 +277,9 @@ MyFrame::MyFrame()
     splitMenu->Append(SPLIT_SETGRAVITY,
                       "Set &gravity\tCtrl-G",
                       "Set gravity of sash");
+    splitMenu->Append(SPLIT_LOCKSASH,
```

Comment:
Shouldn't this be a check item, i.e.
```suggestion
    splitMenu->AppendCheckItem(SPLIT_LOCKSASH,
```

Reply:
Fixed with the next commit.